assets/helpers/externalLinks.js

```diff
@@ -298,6 +303,7 @@ function getWeeklyCheckout(
   cgId: CountryGroupId,
   nativeAbParticipations: Participations,
   optimizeExperiments: OptimizeExperiments,
+  promoCode: Option<string>,
```

Comment:
`getDigitalCheckout` above just takes this value from the `defaultPromos` object would this work here to keep it consistent?

Reply:
`defaultPromos` are always for `GBPCountries` which I _think_ would be ok for now. Ideally I that would be a function that looks up defaults with the active the country code I suppose.

Reply:
Ok, up to you I'm happy with it. We did discuss getting rid of the '10ANNUAL' code and just implementing this as a price change to the annual product but for now this looks fine.